### DISCUSSION TITLE
Quelques modifications pour l'outil d'import.

### DISCUSSION THREAD

**Discussion by gauthierandre** [betagouv/ma-cantine#1064] [Retours utilisateur·rice]
Retour d'un CD : 
1/ Je repends un élément de JB : changer les éléments en anglais pour du français (même si la notice est perçue comme claire).
2/ Pour les colonnes : affiner : Total / Bio / Qualité et durable (hors bio) : préciser HT sur chacun, et peut-être "total des achats" car le mot "total" est équivoque (total de quoi?).

Côté positif : outil qui semble simple d'utilisation (une fois les précisions apportées) et qui va clairement facilité la tâche des chargé.e.s de missions !

**Comment by hfroot** [comment]
Ça me semble clair - j'ai créé un issue pour ça https://github.com/betagouv/ma-cantine/issues/1071

**Comment by valeriemerle** [comment]
Cette discussion rejoint une autre dans laquelle j'ai fais ds retours sur la fonction import. Il ne faudrait garder qu'une discussion sur ce sujet sinon on risque de se perdre?

**Reply by alemangui** [reply]
Tu fais référence à ce sous-commentaire ? https://github.com/betagouv/ma-cantine/discussions/1003#discussioncomment-2220933
